src/modules/projects/components/tables/ProjectExternalSubmissionsTable.tsx

```diff
@@ -101,20 +101,22 @@ const ProjectExternalSubmissionsTable = ({
           ),
         },
         ...defs,
-        {
-          header: 'Action',
-          render: ({ id }: ExternalFormSubmissionSummaryFragment) => (
-            <Button
-              variant='outlined'
-              onClick={() => setModalOpenId(id)}
-              disabled={bulkLoading}
-            >
-              View
-            </Button>
-          ),
-        },
       ];
     },
+    []
+  );
+
+  const getTableRowActions = useCallback(
```

Comment:
Changing this table is out of scope of the ticket (not in the spreadsheet), but it was an especially easy example to update to the new pattern since it already used an Action Button column, so I pulled it in.